deployment/network-operator/Chart.yaml

```diff
@@ -20,10 +20,6 @@ dependencies:
     name: sriov-network-operator
     repository: ''
     version: 0.1.0
-  - condition: nicConfigurationOperator.enabled
```

Comment:
It's good to delete. this in general but we meting this is mentioned in our documentation so it seems we need to support it at lease for one release as a deprecated way to deploy NIC Configuration Operator

[1] https://docs.nvidia.com/networking/display/kubernetes2501/customizations/helm.html

Reply:
Dropped the commit with deletion of the nic operator helm chart. Let's add the deprecation message in a separate commit with doc updates